### PR TITLE
check the round on returned beacons

### PR DESCRIPTION
**Pull request by CluEleSsUK**

### CODE REVIEW

File: lib/index.ts [hunk 1]
```diff
@@ -77,7 +77,9 @@ export async function fetchBeacon(client: ChainClient, roundNumber?: number): Pr
         beacon = await client.get(roundNumber)
     }
 
-    return validatedBeacon(client, beacon)
+    const expectedRound = roundAt(Date.now(), await client.chain().info())
```
Comment (lourkeur) [inline lib/index.ts:80]: This should take into account the case where roundNumber is provided.
```suggestion
    const expectedRound = roundNumber || roundAt(Date.now(), await client.chain().info())
```

Comment (CluEleSsUK) [reply lib/index.ts:80]: oops yeah I refactored this out by accident hehe

**Review by AnomalRoil — APPROVED** [review]

File: package.json [hunk 2]
```diff
@@ -29,6 +29,7 @@
     "typescript-eslint": "^0.0.1-alpha.0"
   },
   "dependencies": {
+    "@babel/traverse": "^7.23.2",
```
Comment (imcotton) [inline package.json:32]: Why this dependency `@babel/traverse` has been added here? Seems not been used anywhere?

/cc @CluEleSsUK

Comment (CluEleSsUK) [reply package.json:32]: Great question - I have no idea, it seems unused; I will remove it

**Pull request merged**